### PR TITLE
modified game, inventory, selectplant

### PR DESCRIPTION
**Pull request by Andrea-Laserna**
- added getPlants() and getBoughtItems in Game

- added conditions to Inventory and SelectPlant

### CODE REVIEW

File: src/Plantiti/Game.java [hunk 1]
```diff
@@ -72,9 +72,11 @@ public boolean useBoughtItem(BuyableItem itemUsed) {
         return false;
     }
 
+    //NEWLY MODIFIED LIST PLANTS AS FOR LOOP
```
Comment (DuskyElf) [inline src/Plantiti/Game.java:75]: Should remove these comments before merging to the upstream

File: src/Plantiti/Options/SelectPlant.java [hunk 2]
```diff
@@ -5,19 +5,28 @@
...
-        game.listPlants();
-        System.out.println("Enter the Index of plant to select");
 
-        Scanner scanner = new Scanner(System.in);
-        int userInput = scanner.nextInt();
-
-        boolean success = game.selectPlant(userInput);
-        if (!success) {
-            System.out.println("Invalid Index: Try again ;P");
+        //CONDITION NEW ADDITION
```
Comment (DuskyElf) [inline src/Plantiti/Options/SelectPlant.java:9]: This comment should be removed

File: src/Plantiti/Options/Inventory.java [hunk 3]
```diff
@@ -3,9 +3,18 @@
 
 public class Inventory extends Option {
     public void runOption(Game game) {
-        System.out.println("Your Plants:");
-        game.listPlants();
-        System.out.println("Your Items:");
-        game.listItems();
+        if (game.getPlants().isEmpty()) {
```
Comment (DuskyElf) [inline src/Plantiti/Options/Inventory.java:6]: nice 👍

File: src/Plantiti/Options/SelectPlant.java [hunk 4]
```diff
@@ -5,19 +5,28 @@
...
+        if (game.getPlants().isEmpty()){
+            System.out.println("You don't have plants at the moment. Type 'Shop' to buy a plant.");
             return;
+        } else {
```
Comment (DuskyElf) [inline src/Plantiti/Options/SelectPlant.java:13]: don't need this else block, the previous if condition already returns from the function

File: src/Plantiti/Game.java [hunk 5]
```diff
@@ -102,4 +104,15 @@ public void addItem(BuyableItem item) {
     public void addPlant(Plant plant) {
         plants.add(plant);
     }
+
+    //NEW ADDITION GETPLANTS METHOD
```
Comment (DuskyElf) [inline src/Plantiti/Game.java:108]: same comment removal needed

File: src/Plantiti/Game.java [hunk 6]
```diff
@@ -102,4 +104,15 @@ public void addItem(BuyableItem item) {
...
+    public ArrayList<Plant> getPlants() {
+        return plants;
+    }
+
+    //NEW ADDITION RETRIEVE BOUGHT ITEMS
```
Comment (DuskyElf) [inline src/Plantiti/Game.java:113]: same comment removal needed

**Review by DuskyElf — CHANGES_REQUESTED** [review]
Amazing work, a few changes are requested. 👍

File: src/Plantiti/Options/SelectPlant.java (hunk 2 above)
Comment (Andrea-Laserna) [reply src/Plantiti/Options/SelectPlant.java]: done

File: src/Plantiti/Options/SelectPlant.java (hunk 4 above)
Comment (Andrea-Laserna) [reply src/Plantiti/Options/SelectPlant.java]: done

File: src/Plantiti/Options/Inventory.java (hunk 3 above)
Comment (Andrea-Laserna) [reply src/Plantiti/Options/Inventory.java:6]: thank u

**Review by DuskyElf — APPROVED** [review]
Amazing work,. Approved 👍

**Pull request merged**